src/DssSpell.sol

```diff
@@ -43,8 +49,34 @@ contract DssSpellAction is DssAction {
     uint256 internal constant WAD     = 10 ** 18;
     uint256 internal constant RAY     = 10 ** 27;
 
+    function _sub(uint256 x, uint256 y) internal pure returns (uint256 z) {
+        require((z = x - y) <= x, "sub-underflow");
+    }
+
+    uint256 internal constant PSM_ZERO_BASIS_POINTS = 0;
```

Comment:
i don't think this is necessary

```suggestion
    uint256 internal constant PSM_ZERO_BASIS_POINTS = 0;
```

Reply:
OK will remove was just following previous patterns